docs/site/Decorators.md

```diff
@@ -22,647 +22,7 @@ tie into an API.
-
-2. Define your own repository without using the datasource juggler bridge, and
-   use an ORM/ODM of your choice.
-
-```ts
-// Missing example here
-// Will be provided in Model.md
-// refer to [example code](https://github.com/strongloop/loopback-next-example/blob/master/services/account-without-juggler/repositories/account/models/Account.ts)
-```
-
-#### Property Decorator
-
-Syntax: `@property(definition?: PropertyDefinition)`
-
-The property decorator defines metadata for a property on a Model definition.
-The format of property definitions can be found in
-[Property definitions](https://loopback.io/doc/en/lb2/Model-definition-JSON-file.html#properties)
-
-For usage examples, see [Define Models](Repositories.md#define-models)
-
-### Repository Decorator
-
-Syntax:
-
-[`@repository(modelOrRepo: string | Class<Repository<Model>> | typeof Entity, dataSource?: string | juggler.DataSource)`](http://apidocs.loopback.io/@loopback%2fdocs/repository.html#repository)
-
-This decorator either injects an existing repository or creates a repository
-from a model and a datasource.
-
-The injection example can be found in
-[Repository#controller-configuration](Repositories.md#controller-configuration)
-
-To create a repository in a controller, you can define your model and datasource
-first, then import them in your controller file:
-
-```ts
-// src/controllers/todo.controller.ts
-import {Todo} from '../models';
-import {db} from '../datasources/db.datasource';
-import {repository, EntityCrudRepository} from '@loopback/repository';
-
-export class TodoController {
-  @repository(Todo, db)
-  todoRepo: EntityCrudRepository<Todo, number>;
-  // ...
-}
-```
-
-If the model or datasource is already bound to the app, you can create the
-repository by providing their names instead of the classes. For example:
-
-```ts
-// with `datasource` and `Todo` already defined.
-app.bind('datasources.db').to(db);
-app.bind('models.Todo').to(Todo);
-
-export class TodoController {
-  @repository('Todo', 'db')
-  repository: EntityCrudRepository<Todo, number>;
-  // etc
-}
-```
-
-### Relation Decorators
-
-_This feature has not yet been released in alpha form. Documentation will be
-added here as this feature progresses._
-
-The relation decorator defines the nature of a relationship between two models.
-
-#### Relation Decorator
-
-Syntax: `@relation`
-
-Register a general relation.
-
-_This feature has not yet been released in alpha form. Documentation will be
-added here as this feature progresses._
-
-#### Specific Relation Decorator
-
-Syntax:
-
-- `@belongsTo`
-- `@hasOne`
-- `@hasMany`
-- `@embedsOne`
-- `@embedsMany`
-- `@referencesOne`
-- `@referencesMany`
-
-Register a specific relation.
-
-_This feature has not yet been released in alpha form. Documentation will be
-added here as this feature progresses._
+- [Route Decorators](Decorators_route.md)
+- [Dependency Injection Decorator](Decorators_inject.md)
+- [Authentication Decorator](Decorators_auth.md)
```

Comment:
I wonder if it's better for us to use `Decorators_authenticate` and `Decorators_repository` as that is the full name of the actual decorator being described by the doc page.

Reply:
Renamed